src/components/Sidebar/NavContent.jsx

```diff
@@ -31,6 +31,12 @@ function NavContent() {
     if ((currentPath.indexOf("bonds") >= 0 || currentPath.indexOf("choose_bond") >= 0) && page === "bonds") {
       return true;
     }
+    if (
+      (currentPath.indexOf("33-together") >= 0 || currentPath.indexOf("33-together") >= 0) &&
```

Comment:
```suggestion
      currentPath.indexOf("33-together") >= 0 &&
```
Don't need to check the same check twice, that was likely an artifact from being copied from previous lines.

Reply:
Nice catch, will update!